Source/Core/DolphinQt/CheatSearchWidget.cpp

```diff
@@ -0,0 +1,523 @@
+              .arg(QString::fromStdString(fmt::format("0x{0:08x}", m1.m_start)))
+              .arg(QString::fromStdString(fmt::format("0x{0:08x}", m1.m_start + m1.m_length - 1)));
+      break;
+    }
+    default:
+      ranges = tr("%1 memory ranges").arg(range_count);
+      break;
+    }
+
+    QString space;
+    switch (m_session->GetAddressSpace())
+    {
+    case PowerPC::RequestedAddressSpace::Effective:
+      space = tr("Address space by CPU state");
+      break;
+    case PowerPC::RequestedAddressSpace::Physical:
+      space = tr("Physical address space");
+      break;
+    case PowerPC::RequestedAddressSpace::Virtual:
+      space = tr("Virtual address space");
+      break;
+    default:
+      space = tr("Unknown address space");
+      break;
+    }
+
+    QString type;
+    switch (m_session->GetDataType())
+    {
+    case Cheats::DataType::U8:
+      type = tr("8-bit Unsigned Integer");
+      break;
+    case Cheats::DataType::U16:
+      type = tr("16-bit Unsigned Integer");
+      break;
+    case Cheats::DataType::U32:
+      type = tr("32-bit Unsigned Integer");
+      break;
+    case Cheats::DataType::U64:
+      type = tr("64-bit Unsigned Integer");
+      break;
+    case Cheats::DataType::S8:
+      type = tr("8-bit Signed Integer");
+      break;
+    case Cheats::DataType::S16:
+      type = tr("16-bit Signed Integer");
+      break;
+    case Cheats::DataType::S32:
+      type = tr("32-bit Signed Integer");
+      break;
+    case Cheats::DataType::S64:
+      type = tr("64-bit Signed Integer");
+      break;
+    case Cheats::DataType::F32:
+      type = tr("32-bit Float");
+      break;
+    case Cheats::DataType::F64:
+      type = tr("64-bit Float");
+      break;
+    default:
+      type = tr("Unknown data type");
+      break;
+    }
+    QString aligned = m_session->GetAligned() ? tr("aligned") : tr("unaligned");
+    session_info_label->setText(tr("%1, %2, %3, %4").arg(ranges).arg(space).arg(type).arg(aligned));
+  }
+
+  auto* value_layout = new QHBoxLayout();
+
+  auto* instructions_label = new QLabel(tr("Keep addresses where value in memory"));
+  value_layout->addWidget(instructions_label);
+
+  m_compare_type_dropdown = new QComboBox();
+  m_compare_type_dropdown->addItem(tr("is equal to"),
+                                   QVariant::fromValue(Cheats::CompareType::Equal));
+  m_compare_type_dropdown->addItem(tr("is not equal to"),
+                                   QVariant::fromValue(Cheats::CompareType::NotEqual));
+  m_compare_type_dropdown->addItem(tr("is less than"),
+                                   QVariant::fromValue(Cheats::CompareType::Less));
+  m_compare_type_dropdown->addItem(tr("is less than or equal to"),
+                                   QVariant::fromValue(Cheats::CompareType::LessEqual));
+  m_compare_type_dropdown->addItem(tr("is more than"),
+                                   QVariant::fromValue(Cheats::CompareType::More));
```

Comment:
More important with the UI strings, use "Greater" rather than "More" as that is how the comparison operations are called in both programming languages and debuggers.